rllib/core/torch/torch_rl_module.py

```diff
@@ -0,0 +1,18 @@
+import torch
+
+from ray.rllib.core.rl_module import RLModule
+
+
+class TorchRLModule(torch.nn.Module, RLModule):
+    def __init__(self, config):
+        super().__init__()
+        self.config = config
+
+    def forward(self, batch, **kwargs):
```

Comment:
RLModule doesn't have forward() though? only __call__, forward_train, and forward_inference.

Reply:
ok so the reason I did this is because I think the DDP wrapper requires that forward is implemented by the module that is being optimized. Although its possible we might be able to get away with just implementing __call__